include/deal.II/multigrid/mg_transfer_global_coarsening.h

```diff
@@ -407,22 +407,10 @@ class MGTwoLevelTransfer<dim, LinearAlgebra::distributed::Vector<Number>>
   mutable LinearAlgebra::distributed::Vector<Number> vec_coarse;
 
   /**
-   * Constraint-entry indices for performing manual
-   * constraint_coarse.distribute_local_to_global().
+   * TODO
```

Comment:
Can you write a short documentation here?

Reply:
Done!